shell/browser/notifications/notification.h

```diff
@@ -30,14 +45,24 @@ struct NotificationOptions {
   bool silent;
   GURL icon_url;
   SkBitmap icon;
+  GURL image_url;
+  SkBitmap image;
   bool has_reply;
   std::u16string timeout_type;
   std::u16string reply_placeholder;
   std::u16string sound;
   std::u16string urgency;  // Linux
+  std::u16string data;     // feat: Support Notification.data property
   std::vector<NotificationAction> actions;
   std::u16string close_button_text;
   std::u16string toast_xml;
+  bool is_persistent;        // feat: Correct processing for the persistent
+                             // notification without actions
+  bool require_interaction;  // feat: Configuration toast show time
+                             // according to
+                             // Notification.requireInteraction property
+  bool should_be_presented;  // feat: display toast according to
+                             // Notification.renotify
```

Comment:
These comments are unnecessary IMO

Reply:
Ok, do not have any objections to remove them

Reply:
updated: 6556c323521deafba5fd68f5ca1d8a8bb2ea2fa4